azure-pipelines.yml

```diff
@@ -79,17 +78,17 @@ jobs:
       inputs:
         artifactName: linux-release
         targetPath: $(Build.StagingDirectory)/release-artifacts
-      condition: not(eq(variables['skip-github'], 'TRUE'))
+      condition: and(succeeded(), not(eq(variables['skip-github'], 'TRUE')))
```

Comment:
I’m starting to wonder whether I should write a linter for azure pipelines configs :slightly_smiling_face: